crates/typst-kit/Cargo.toml

```diff
@@ -0,0 +1,82 @@
+[package]
+name = "typst-kit"
+description = "Common utilities for Typst tooling."
+version = { workspace = true }
+rust-version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
+readme = { workspace = true }
+
+[dependencies]
+typst = { workspace = true }
+typst-assets = { workspace = true, optional = true }
+typst-timing = { workspace = true }
+ecow = { workspace = true }
+env_proxy = { workspace = true, optional = true }
+dirs = { workspace = true, optional = true }
+flate2 = { workspace = true, optional = true }
+fontdb = { workspace = true, optional = true }
+native-tls = { workspace = true, optional = true }
+once_cell = { workspace = true }
+tar = { workspace = true, optional = true }
+ureq = { workspace = true, optional = true }
+
+# Explicitly depend on OpenSSL if applicable, so that we can add the
+# `openssl/vendored` feature to it if `vendor-openssl` is enabled.
+[target.'cfg(not(any(target_os = "windows", target_os = "macos", target_os = "ios", target_os = "watchos", target_os = "tvos")))'.dependencies]
+openssl = { workspace = true }
+
+[features]
+default = [
+  "fonts",
+  "packages",
+  "project",
+]
+
+# Add font loading utilitites
+fonts = [
+  "dep:fontdb",
+  "fontdb/memmap",
```

Comment:
can/do these typicaly also use `dep:` or not? I don't really know.

Reply:
Using `dep:` for an optional dependency means that there's no implicit feature created for it (https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies).

Reply:
What I meant was whether `"dep:fontdb/memmap"` is a thing, but on second thought it probably makes no sense, since the implicit feature is already not created.